sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -16,6 +17,8 @@ import io.sentry.hints.SessionEndHint
 import io.sentry.hints.SessionStartHint
 import io.sentry.protocol.SentryId
 import io.sentry.protocol.User
+import org.mockito.ArgumentCaptor
```

Comment:
you can use https://github.com/nhaarman/mockito-kotlin/wiki/Mocking-and-verifying#argument-captors
the extension methods are nicer

Reply:
Yes definitely, thanks for pointing that out. 💯